### PR TITLE
Add size formats as described in issue #102

### PR DESCRIPTION
**Pull request by fumikito**
- Add new formats(Kib, KB and etc).
- Add help comment.
- Add scenario test.

Hope it helps!

### CODE REVIEW

File: src/DB_Command.php [hunk 1]
```diff
@@ -659,6 +659,15 @@ public function tables( $args, $assoc_args ) {
 	 *  - mb (megabytes)
 	 *  - gb (gigabytes)
 	 *  - tb (terabytes)
+	 *  - B   (ISO Byte setting, with no conversion)
+	 *  - KB  (ISO Kilobyte setting, with 1 kB  = 1,000 B)
```
Comment (schlessera) [inline src/DB_Command.php]: Case is wrong in explanation. `kB` => `KB`

File: src/DB_Command.php [hunk 2]
```diff
@@ -659,6 +659,15 @@ public function tables( $args, $assoc_args ) {
...
+	 *  - KiB (ISO Kibibyte setting, with 1 KiB = 1,024 B)
+	 *  - MB  (ISO Megabyte setting, with 1 MB  = 1,000 kB)
```
Comment (schlessera) [inline src/DB_Command.php]: Case is wrong in explanation. `kB` => `KB`

**Comment by fumikito** [comment]
@schlessera Thank you for your review. Sorry for my typo and now they are fixed!

**Comment by schlessera** [comment]
Thanks for the pull-request, @fumikito!

**Pull request merged**